### PR TITLE
feat(fundamentals): add all the fundamentals workshops

### PR DESCRIPTION
**Pull request by nimbinatus**
part of pulumi/devrel-team#391

### CODE REVIEW

File: fundamentals/csharp/lab-2/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+The application we'll be running on our infrastructure is in the [pulumi/tutorial-pulumi-fundamentals repo](https://github.com/pulumi/tutorial-pulumi-fundamentals) in the `app/` directory. Examine the Dockerfiles in each directory.
+
+<details>
+<summary><b>Question:</b> What's the Dockerfile in the `backend` directory doing?</summary>
+
+<br/>
+<b>Answer:</b> This Dockerfile copies the REST backend into the Docker filesystem, installs the dependencies, and builds the image. Note that port 3000 must be open on your host machine.
+</details>
+<br/>
```
Comment (cnunciato) [inline fundamentals/csharp/lab-2/README.md:15]: This is neat. We should do more of this. :)

File: fundamentals/go/lab-2/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+<summary><b>Question:</b> What's the Dockerfile in the `backend` directory doing?</summary>
+
+<br/>
+<b>Answer:</b> This Dockerfile copies the REST backend into the Docker filesystem, installs the dependencies, and builds the image. Note that port 3000 must be open on your host machine.
+</details>
+<br/>
+
+## Build your Docker Image with Pulumi
+
+<!-- add deps -->
```
Comment (cnunciato) [inline fundamentals/go/lab-2/README.md:19]: Is this a still-to-do? (Only asking in case it is. 😄)

**Review by cnunciato — APPROVED** [review]
These changes all look good to me. 👍

File: fundamentals/go/lab-2/README.md (hunk 2 above)
Comment (nimbinatus) [reply fundamentals/go/lab-2/README.md:19]: derp 😆 fixing!

File: fundamentals/csharp/lab-2/README.md (hunk 1 above)
Comment (nimbinatus) [reply fundamentals/csharp/lab-2/README.md:15]: thanks!

**Pull request merged**